src/deb/helios-agent/control

```diff
@@ -3,7 +3,7 @@ Version: [[version]]
 Section: non-free/net
 Priority: optional
 Architecture: all
-Depends: java7-runtime, helios-services (= [[version]])
+Depends: java7-runtime, java8-runtime, helios-services (= [[version]])
```

Comment:
I guess not? I forgot to remove that

On Friday, March 6, 2015, Ryan Culbertson <notifications@github.com> wrote:

> In src/deb/helios-agent/control
> <https://github.com/spotify/helios/pull/384#discussion_r25987314>:
>
> > @@ -3,7 +3,7 @@ Version: [[version]]
> >  Section: non-free/net
> >  Priority: optional
> >  Architecture: all
> > -Depends: java7-runtime, helios-services (= [[version]])
> > +Depends: java7-runtime, java8-runtime, helios-services (= [[version]])
>
> Any reason to require 7 as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spotify/helios/pull/384/files#r25987314>.
>


-- 
David Xia
Software Engineer
dxia@spotify.com